examples/dex/README.md

```diff
@@ -40,7 +44,8 @@ kubectl -n authorino apply -f ./examples/echo-api-protection.yaml
 
 Add dex to the cluster:
 ```shell
-kubectl -n authorino create -f examples/dex.yaml
+kubectl create -f examples/dex/crd.yaml
```

Comment:
Shouldn't installing the CRD responsibility of the owner of the CRD (usually an operator)? Shouldn't be required the operator to be installed up  and running?

Reply:
There's no operator in this case. This CRD is the data structure dex uses to store its state in etcd, alternatively to an actual database system.

Reply:
It is very weird, but ok

Reply:
Yeah. I went with dex v2.26 because the version we were using before (v2.10) didn't have the OAuth2 UserInfo endpoint. dex introduced it in v2.17.

When the example was based still on v2.10, dex was configured to store data in a sqlite3 db. Unfortunately that didn't work when I upgraded to v2.26. I think it's related to our Kind cluster and permissions. Therefore I switched dex's storage to "kubernetes", which means it will store its data in CRs.

Right now, I have no plans to deploy any dex operator only for this "Trying Authorino with dex" example. Unless it was something super easy to do, but I honestly couldn't even find anything about a "dex operator" in their documentation, nor in OperatorHub. So we're hosting the CRDs, whose version should be paired with image version of dex that we are using.

For reference:
- https://dexidp.io/docs/kubernetes/#deploy-the-dex-server
- https://dexidp.io/docs/storage/#kubernetes-custom-resource-definitions-crds

Reply:
For devtesting purposes, it should not be a big deal.  :+1: